apps/giphy/giphy.star

```diff
@@ -0,0 +1,132 @@
+"""
+Applet: Giphy
+Summary: Displays Giphy gifs
+Description: Displays a random gif based on a search query. Powered by Giphy.com.
+Author: Ricky Smith (DigitallyBorn)
+"""
+
+load("cache.star", "cache")
+load("encoding/base64.star", "base64")
+load("encoding/json.star", "json")
+load("http.star", "http")
+load("random.star", "random")
+load("render.star", "render")
+load("schema.star", "schema")
+load("secret.star", "secret")
+
+SEARCH_URL = "https://api.giphy.com/v1/gifs/search?api_key={}&q={}&limit=25&offset=0&rating={}&lang=en"
+API_TOKEN_ENCRYPTED = """
```

Comment:
This secret, when decrypted, seems to be wrapped in json, like: `{"api_token":"..."}`. I think you only want the value of that key-value pair in there.

Reply:
Yup ... I thought I was going to have another value to include in the secrets, so I created it as json ... ended up not needing it. Should be fixed now.